Library/Homebrew/global.rb

```diff
@@ -49,6 +49,8 @@ module Homebrew
   extend FileUtils
 
   DEFAULT_PREFIX ||= "/usr/local".freeze
+  DEFAULT_CELLAR = "#{DEFAULT_PREFIX}/Cellar".freeze
+  DEFAULT_REPOSITORY = "#{DEFAULT_PREFIX}/Homebrew".freeze
```

Comment:
Just to be certain, the use of `=` over `||=` is deliberate here? (I expect it is.)

Reply:
Yes, it is. The use of `||=` is due to the definition of this same variable at
https://github.com/Homebrew/brew/blob/7e63dfb55656591dfeb5d0b98314090f81ed3304/Library/Homebrew/os/linux/global.rb#L2